app/main.py

```diff
@@ -1,8 +1,51 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class = comfort_class if 1 <= comfort_class <= 7 else 1
+        self.clean_mark = clean_mark if 1 <= clean_mark <= 10 else 1
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 distance_from_city_center: float,
+                 clean_power: int,
+                 average_rating: float,
+                 count_of_ratings: int) -> None:
+        self.distance_from_city_center = (
+            distance_from_city_center
+            if 1.0 <= distance_from_city_center <= 10.0
+            else 1.0
+        )
+        self.clean_power = clean_power
+        self.average_rating = average_rating if 1.0 <= average_rating <= 5.0 \
```

Comment:
As per styling guidelines we  don't use backslash for breaking lines. You can do the same way you did it on line 17-21